internal/app/handlers/shortener_handler.go

```diff
@@ -9,6 +10,8 @@ import (
 	"regexp"
 )
 
+var regexMustCompileURL = regexp.MustCompile(`(http|https)://\S+`)
```

Comment:
👍 за вынос компиляции регэксп выражения на уровень пакета! Единственное мне кажется нет особой ценности в именовании переменной `regexMustCompileURL`, я бы предложил упростить: 
 
```suggestion
var regexURL = regexp.MustCompile(`(http|https)://\S+`)
```